app/server_app_adapters.go

```diff
@@ -139,6 +139,13 @@ func (s *Server) RunOldAppInitialization() error {
 	}
 	s.Router = s.RootRouter.PathPrefix(subpath).Subrouter()
 
+	// FakeApp: remove this when we have the ServePluginRequest and ServePluginPublicRequest migrated in the server
```

Comment:
Does this mean https://github.com/mattermost/mattermost-server/pull/14803 has to wait until this is fixed, or that can still go in ?

Reply:
You can go in, this doesn't change anything, is just a bit ugly, and we have to take care of that. but nothing that is going to change anything fundamental.

Reply:
Filed https://mattermost.atlassian.net/browse/MM-26020.